pkg/piecestore/rpc/server/psdb/psdb.go

```diff
@@ -247,3 +274,75 @@ func (db *DB) DeleteTTLByID(id string) error {
 	}
 	return err
 }
+
+// AddMIB adds MIB into database by date
+func (db *DB) AddBwUsageTbl(size, unixtimenow int64) (err error) {
+	defer db.locked()()
+	t := time.Now()
```

Comment:
Why is both `t` used and `now` passed in?

Reply:
Thanks you sir.. Its actually not needed. 
1. Modified to the arg 'unixtimenow' to use time.Time and made functionality change accordingly.